community/self-hosting-novu/introduction.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: 'Introduction'
+description: ''
+icon: 'code'
+---
+
+Self-hosting Novu involves setting up and maintaining your communication infrastructure using your own servers.
+While this offers full control and customization,
+some features which are available only for Novu’s cloud-managed won't be available in a self-hosted environment.
+
+## Minimum System Requirements
```

Comment:
The phrasing of this section feels a bit hasty. How about adding a section for each scenario and then a list of the hardware requirements for each?

1. Host each Novu Service in a separated VM
2. Host all Novu services in a single VM
3. Redis
4. Mongo
5. ...

Reply:
Sure, I used the old page. Let me rewrite this